guides/common/assembly_configuring-provisioning-resources.adoc

```diff
@@ -1,4 +1,6 @@
+ifdef::katello,orcharhino,satellite[]
 include::modules/con_configuring-provisioning-resources.adoc[]
```

Comment:
question: why are you hiding this for foreman without katello? The concept only contains a heading.

Reply:
I think I messed up the PR, will update.